zellij-server/src/plugins/unit/plugin_tests.rs

```diff
@@ -6191,6 +6191,7 @@ pub fn unblock_input_plugin_command() {
     std::thread::sleep(std::time::Duration::from_millis(500));
 
     let _ = plugin_thread_sender.send(PluginInstruction::CliPipe {
+        pane_id: None,
```

Comment:
Let's maybe make one of these a `Some` for the sake of completeness in the tests?